src/core/execute.c

```diff
@@ -1130,6 +1143,9 @@ static int apply_address_families(const ExecContext *c) {
 
         assert(c);
 
+        if (skip_seccomp_unavailable("RestrictAddressFamilies"))
```

Comment:
nitpick: in the docs we usually suffix unit file settings with "=", to indicate that they are unit file settings... I think we should stick to that in log messages too.